### ISSUE TITLE
More decorative tiles and furniture

### ISSUE THREAD

**Issue by BenCheung0422** [MinicraftPlus/minicraft-plus-revived#539]
**Is your feature request related to a problem? Please describe.**
As there could be more fancy decorations in the game, more decorative tiles and furniture could be added.

**Describe the solution you'd like**
For rock varieties, there could be granite, cobblestone, obsidian, sandstone, basalt, tuff, andesite, diorite, calcite, etc. For ore varieties, there could be coal and any other useful metals or rocks. Also, there could be different abundances embedded in rocks instead of dedicated as their own tiles. There could also be different wood types in the game, making planks more colorful.
As of furniture, there should be more decorative furniture, like lights, glasses, house furniture, statues, etc. There could also be processed decorative tiles made up of such materials.

**Comment by JamesTDG** [comment]
Additionally, existing textures should be updated to provide designs that provide different textures beyond just color swapping

**Comment by BenCheung0422** [comment]
I agree on this, it can improve the visual experience and abundance.
